mantidproject/mantid
# Attach muon plotting to GUI

**Description of work.**
This PR improves the plotting within the Muon analysis and Frequency domain analysis GUIs by making the plotting window a dockable widget. This change is made to improve the user experience, particularly when working on laptops. This change is implemented through a QtDockWidget.

**To test:**
(1) Load up the Muon analysis GUI
(2) Load a run data (e.g 2223)
(3) Check that plot is created
(4) Change group / pair to bwd, check that the plot updates correctly
(5) Change plot type to counts and check that the plot updates correctly
(6) Move the new dock widget about, ensuring that no issues are encountered.


Fixes #26883. 




---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
